### DISCUSSION TITLE
Git training step by step

### DISCUSSION THREAD

**Discussion by Sepehrsalimian** [Sepehrsalimian/WebStarter#26] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/b96c5cd1-e656-4bd6-8bca-9c422d9ab7da)
Here is the latest update on committing!
@BCFCODE

**Comment by BCFCODE** [comment]
I can't see any of them on my side—they’re all on your local machine. Push your changes (git push) so I can pull them (git pull) and check out your reports (commits). @Sepehrsalimian

**Reply by Sepehrsalimian** [reply]
Now, you'll be able to check it.
@BCFCODE

**Comment by BCFCODE** [comment]
#Task 
Tell me in detail why we use "git push"
@Sepehrsalimian

**Reply by Sepehrsalimian** [reply, marked as answer]
Git push tells Git you want to transfer your commits from your
**local repository** to a **remote repository** such as GitHub.
@BCFCODE

**Comment by BCFCODE** [comment]
BCFCODEteam convention: Make your Q/As specific. That means framing the question so that the subject of the topic is the actual question, and when you achieve the answer, the person who marks it closes it. @Sepehrsalimian
